cmd/util/compress.go

```diff
@@ -0,0 +1,588 @@
+package util
```

Comment:
maybe it would make sense to write the commit_id of the btcd tree you took it from? so that people can easily know approximately how old it is

Reply:
good idea